edit_test.go

```diff
@@ -306,3 +323,31 @@ func TestBadDelimiterError(t *testing.T) {
 		t.Errorf("badDelimiterError is %v; expected %v", got, want)
 	}
 }
+
+// MockFrame is a mock implementation of a frame.Frame that does nothing.
+type MockFrame struct{}
```

Comment:
I've thought about mocking `draw.Display` (after changing it into an interface). Perhaps if we mock `draw.Display`, `draw.Image`, etc., we wouldn't need to mock `Frame`? Changing `draw.Display` to an interface is not a small change, but I may work on this.

Reply:
We still want a `Frame` mock. Besides... it was easy. :-)

Reply:
And: a `draw.Display` interface would be great. It would permit way better `Frame` tests if nothing else.